plugin/Gui/AdminPageApiSettings.php

```diff
@@ -172,18 +175,69 @@ private function addFormModelGoogleMapsKey()
 
 	private function addFormModelGoogleBotSettings()
 	{
+		$descriptionDoNotModify = '
+			<div class="do-not-modify">
+				 <p>
+					'.esc_html__("This plugin will not modify the title and description. This enables other plugins to manage those tags.",'onoffice-for-wp-websites').'
+				 </p>
+				 <details>
+					<summary>'.esc_html__('Available custom fields','onoffice-for-wp-websites').'</summary>
+					   <p>'.esc_html__('When this option is active, the plugin makes the following custom fields available in the detail view.'
+				.' These custom fields can be used in SEO plugins to fill out the title and description with the information of the currently shown estate.'
+				." For information on how to use custom fields consult you SEO plugin's documentation.",'onoffice-for-wp-websites').'<br>'.
+			esc_html__('These custom fields are only available in the detail view and on no other page.','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Title (onoffice_title)','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Description (onoffice_description)','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Place (onoffice_place)','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Postal code (onoffice_postal_code)','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Property class (onoffice_property_class)','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Marketing method (onoffice_marketing_method)','onoffice-for-wp-websites').'<br>
+					   - '. esc_html__('Data Record Ref No. (onoffice_id)','onoffice-for-wp-websites').'<br>
```

Comment:
Similar to before, the block after the `summary` can be grouped as a single unit to make translation easier. The list should also be a proper HTML list, but if we can use the Markdown approach explained in the other comment, that would be easy.